README.md

```diff
@@ -0,0 +1,85 @@
+# git-graph
+
+A command line tool to visualize Git history graphs in a comprehensible way, following different branching models.
+
+Exemplary for the GitFlow model, the aim is to have a structured graph like this:
```

Comment:
Maybe just `Example`? Also maybe split and turn the sentences around? "The aim is a structured graph of the branches. An example using the GitFlow model looks like this:"? (The writing course is rearing its head here...) (Also may be make GitFlow a hyperlink.)

Reply:
All done as suggested.